### PR TITLE
Adding Social Media Profile Picture

### PR DESCRIPTION
**Pull request by guiqui**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [ ] you only add one (!) new helper per pull request.
- [ ] you have checked if an open PR already exists.
- [ ] the submitted website is focused on a single, development related issue.
- [ ] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

File: helpers/social-media-profile-picture.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Social Media Profile Picture",
+  "desc": "Create professional profile picture for social media",
+  "url": "https://profile.softcloud.pro/",
+  "tags": [
+    "Images"
+  ],
+  "maintainers": [
+    "Guillermo Quiros"
```
Comment (stefanjudis) [inline helpers/social-media-profile-picture.json:9]: Could you add your GitHub handle here?

**Comment by stefanjudis** [comment]
@guiqui Thanks for contributing. Cool tool! 

I left a tiny comment. 

And I have a feature request. 🙈 I'd love to add a custom gradient.

File: helpers/social-media-profile-picture.json (hunk 1 above)
Comment (guiqui) [reply helpers/social-media-profile-picture.json]: Sure, just the user name, right?

**Comment by stefanjudis** [comment]
Thanks!

**Comment by guiqui** [comment]
> @guiqui Thanks for contributing. Cool tool!
> 
> I left a tiny comment.
> 
> And I have a feature request. 🙈 I'd love to add a custom gradient.

Thank you! Should be quite straightforward, I'm also adding custom backgrounds.

**Pull request merged**